jenkins/custom-ci/generic/motr.groovy

```diff
@@ -16,8 +16,8 @@ pipeline {
     parameters {  
         string(name: 'MOTR_URL', defaultValue: 'https://github.com/Seagate/cortx-motr', description: 'Branch for Motr.')
         string(name: 'MOTR_BRANCH', defaultValue: 'main', description: 'Branch for Motr.')
-        string(name: 'S3_URL', defaultValue: 'https://github.com/Seagate/cortx-s3server', description: 'Branch for S3Server')
-        string(name: 'S3_BRANCH', defaultValue: 'main', description: 'Branch for S3Server')
+        string(name: 'CORTX_RGW_URL', defaultValue: 'https://github.com/Seagate/cortx-rgw', description: 'Branch for S3Server')
+        string(name: 'CORTX_RGW_BRANCH', defaultValue: 'main', description: 'Branch for S3Server')
```

Comment:
Description need to be changed for both parameters

Reply:
Fixed